static/src/stylesheets/module/facia/_item.scss

```diff
@@ -188,14 +188,19 @@ $fc-item-gutter: $gs-gutter / 4;
     }
 
     .u-faux-block-link--hover {
-        background-color: colour(neutral-6);
+        // For touch devices we want to reduce the time before the click event
+        // as much as possible, because browsers on touch devices give feedback
+        // at the end of the click event.
+        @include mq(desktop) {
```

Comment:
means no change for pointer devices at this breakpoint, no feedback that the card is clickable, other than the pointer itself